lib/salus/scanners/semgrep.rb

```diff
@@ -89,6 +90,11 @@ def run
           data = JSON.parse(shell_return.stdout)
           hits = data["results"]
           semgrep_non_fatal_errors = data["errors"]
+          if @config['show_syntax_errors'].to_s == 'false'
```

Comment:
Do we need string comparison here, shouldn't it work without it ?

Reply:
Great catch! removed the `to_s`.